git-remote-mediawiki.perl

```diff
@@ -68,6 +68,11 @@
 for (@tracked_namespaces) { s/_/ /g; }
 chomp(@tracked_namespaces);
 
+# Use subdirectories for subpages
+my $use_subpage_dirs = run_git("config --get --bool remote.${remotename}.subpageDirs");
+chomp($use_subpage_dirs);
```

Comment:
can't we do `chomp run_git(..)` here?

Reply:
@anarcat Have no clue. I just saw such `chomp` usage everywhere, so it's "monkey see monkey do". Again, I'm not a Perl guy, so please feel free to modify it anywhere you see it's not good.

Reply:
yeah nevermind, chomp needs a variable, doing `chomp foo()` does this:

```
Can't modify non-lvalue subroutine call of &main::foo in chomp at t.perl line 6, near ");"
```

so nevermind me. my perl is definitely rusty.